examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -74,7 +74,7 @@
     standardize=False,
     smoothing_fwhm=2.0,
     memory="nilearn_cache",
-    memory_level=1,
```

Comment:
without this we are getting some warnings, that the memory_level is 0 and will be set to 1: I would consider this a bit of a buggy behavior that should fixed